apps/seller/src/styles/index.css

```
@@ -1,2 +1,3 @@
 @import '@dessert/ui/styles';
 @source '../../../../packages/ui/src';
+@import './quill-editor-font.css';
```

Stylelint (17.13.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)